### PR TITLE
Update documentation to point towards hosted nodes rather than shimming service

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

We're confident in the rippled node's ability to serve the demo SDK and want to point SLOHack users to these endpoints.

### CODE REVIEW

**Comment by dpurdy** [comment]
could we get:
xrp.xpring.io
test.xrp.xpring.io

when we can and update this text when we can?

@wliu ? @asana: make me a task?

**Comment by dpurdy** [comment]
this is better than what we have now? but would love to get to some cool hostnames for the cool kids? how can i help?

File: README.md [hunk 1]
```diff
@@ -126,8 +126,8 @@ wallet.verify(message, signature); // true
 ```javascript
 const { XpringClient } = require("xpring-js");
 
-const remoteURL = "grpc.xpring.tech:80";
-const xpringClient = XpringClient.xpringClientWithEndpoint(remoteURL);
+const remoteURL = "3.14.64.116:50051"; // TestNet URL, use 18.222.225.81:50051 for MainNet
+const xpringClient = XpringClient.xpringClientWithEndpoint(remoteURL, true);
```
Comment (amiecorso) [inline README.md]: This is the only xpringClient that is constructed with `xpringClientWithEndpoint`... is that correct or should this also use `new` ?

Comment (keefertaylor) [reply README.md]: Thank you!

**Pull request merged**